e2e/tests/0030-router-smoke-test.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+IFS=$'\n\t'
+set -x
+
+SCRIPT_PATH="$(readlink -f "$(dirname "${BASH_SOURCE[0]}")")"
+source "${SCRIPT_PATH}/../helpers.sh"
+
+cleanup() {
+    oc delete route hello-microshift
+    oc delete service hello-microshift
+    oc delete -f "${SCRIPT_PATH}/assets/hello-microshift.yaml"
+}
+trap cleanup EXIT
+
+gcloud compute firewall-rules update "${INSTANCE_PREFIX}" --allow tcp:22,icmp,tcp:80
```

Comment:
Can we split these test scripts up so it is possible to run them against a local deployment, too?